content/04.network_applications.md

```diff
@@ -1,16 +1,16 @@
 ## Applying Knowledge Graphs to Biomedical Challenges
 
-Knowledge graphs can help researchers tackle many biomedical tasks such as finding new treatments for existing drugs [@doi:10.7554/eLife.26726], aiding efforts to diagnose patients [@arxiv:1611.07012] and predicting associations between diseases and biomolecules [@doi:10.1155/2017/2498957].
-In many cases, solutions rely on representing knowledges graphs in a low dimensional space, which is a process called representation learning.
-This space preserves a knowledge graph's local and/or global structure and can support efforts to apply machine learning methods to make predictions.
-In the next sections we review the unifying techniques that construct this low dimensional space and unifying applications that use this space to solve biomedical problems.
+Knowledge graphs can help researchers tackle many biomedical problems such as finding new treatments for existing drugs [@doi:10.7554/eLife.26726], aiding efforts to diagnose patients [@arxiv:1611.07012] and identfying associations between diseases and biomolecules [@doi:10.1155/2017/2498957].
+In many cases, solutions rely on representing knowledge graphs in a low dimensional space, which is a process called representation learning.
+This space preserves a knowledge graph's local and/or global structure and is straightforward to utilize in efforts that apply machine learning methods to make predictions.
+In the following sections we review methods that construct this low dimensional space (Unifying Relational Learning Techniques) and discuss applications that use this space to solve biomedical problems (Unifying Applications).
```

Comment:
Where does the name "Relational Learning Techniques" come from? Is it widely used in the field?

Reply:
Relational learning came from reviewer 2's comment and from [the paper](https://arxiv.org/pdf/1503.00759.pdf) reviewer 2 mentioned. This term is used in the field to represent methods that work on knowledge graphs (relational data). Upon glancing at the paper, it seems that this term comes from a more statistical point of view. 

After some internal debate, I think the term "representational learning" would be more appropriate as techniques mentioned in this paper focus on low dimensional knowledge graph representations. This term is also widely used in the field, but is more associated with low dimensional embeddings.